alternator/executor.cc

```diff
@@ -2537,7 +2554,10 @@ future<executor::request_return_type> executor::put_item(client_state& client_st
             });
         });
     }
-    co_return co_await op->execute(_proxy, client_state, trace_state, std::move(permit), needs_read_before_write, _stats, _stats.wcu_total[stats::wcu_types::PUT_ITEM]).finally([op, start_time, this] {
+    auto stats = get_stats_from_schema(_proxy, *(op->schema()));
```

Comment:
I think it's better to replace "auto" by the type - it's not obvious that get_stats_from_schema returns a shared pointer, and it's important.

Reply:
Maybe also name it per_table_stats, because it's not obvious below how "stats" and "_stats" differ.

Reply:
sure